src/main/content/_layouts/post.html

```diff
@@ -47,7 +47,11 @@ <h1 id="post_title">{{ page.title | escape }}</h1>
                             {% endif %}
                         {% endfor %}
                     </div>
-                    <a class="author_name" href="{{ page.author_github }}" target="_blank" rel="noopener">{{ page.author }}</a>
+                    {% if page.author_github %}
+                        <a class="author_name" href="{{ page.author_github }}" target="_blank" rel="noopener">{{ page.author }}</a>
+                    {% else %}
+                        <a class="author_name" target="_blank" rel="noopener">{{ page.author }}</a>
```

Comment:
Please change this to be a span element instead: 
`<span class="blog_post_author_name">{{ page.author }}</span>`